student/src/main/java/tds/student/performance/services/impl/TestOpportunityServiceImpl.java

```diff
@@ -161,10 +161,14 @@ public TestConfig startTestOpportunity(OpportunityInstance opportunityInstance,
                     testOpportunity.getClientName(),
                     testOpportunity.getTestId());
 
+            Boolean isMsb = configurationDao.isMsb(
```

Comment:
Does this have to be the Object boolean or could it be the primitive?  I understand the current code base likes to use Objects instead of primitives